pkg/watch/health.go

```diff
@@ -0,0 +1,240 @@
+package watch
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"os/exec"
+	"time"
+
+	"github.com/square/p2/pkg/health"
+	"github.com/square/p2/pkg/kp"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/util/net"
+)
+
+// number of milliseconds between reality store checks
+const POLL_KV_FOR_PODS = 4000
+
+// number of milliseconds between health checks
+const HEALTHCHECK_INTERVAL = 200
```

Comment:
We will probably want to both increase this interval (a healthcheck every 200 millis is pretty aggressive) and enable some form of backoff.